src/_actions/paymentActions.js

```diff
@@ -28,8 +33,17 @@ const makePayment = (phone, amount) => async (dispatch) => {
 const confirmPayment = (requestID) => async (dispatch) => {
   dispatch({ type: CONFIRM_PAYMENT_REQUEST, payload: { requestID } });
   try {
-    const { data } = await axios.post("/query", { requestID });
-    dispatch({ type: CONFIRM_PAYMENT_SUCCESS, payload: data });
+    await axios.post("/query", { requestID })
+    .then(response=>{
+    // dispatch({ type: CONFIRM_PAYMENT_SUCCESS, payload: response.data });
+    Cookie.set('confirmPaid', JSON.stringify(response.data),{
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    Cookie.set("confirmPaid", JSON.stringify(response.data),{
```